cmd/blackfire.go

```diff
@@ -0,0 +1,90 @@
+// Copyright © 2017 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"errors"
+	//"bufio"
+
+	"github.com/spf13/cobra"
+)
+
+// startBlackfireCmd represents the startXdebug command
+var startBlackfireCmd = &cobra.Command{
+	Use:   "blackfire",
+	Short: "Checks if environment is correctly set for Blackfire",
+	Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("	---> Check Blackfire requirement")
+		fmt.Println("		---> Ensuring Blackfire environment variables are defined")
```

Comment:
should we add a message to say that your containers will need the blackfire probe installed as well for this to work?